### ISSUE TITLE
alignment preprocessing - align all maps in submission to reference

### ISSUE THREAD

**Issue by geoffwoollard** [flatironinstitute/Cryo-EM-Heterogeneity-Challenge-1#91]
Problem:
- Currently there is a global alignment for a submission, in the sense that one common rotation is applied to all 80 maps.
- However the motion of the 80 maps could have a difference "frame of reference". In the sense that some motion and the global alignment of each map leak into each other. For instance, consider the two scenarios
  - you align the "middle" region, and the "left" and "right" domain (look like butterfly wings) flap evenly
  - if you align the "left" domain, then the "right" domain would move twice as much
- There are no guarantees for how the heterogeneity moves around. Visually it seems to be quite similar to the ground truth's frame of reference, but this might just be because the difference is subtle.

Solution:
- include option to align all 80 maps individually to the reference map

**Comment by DSilva27** [comment]
This should be pretty simple to implement. Simply check out [_preprocessing/align_utils.py](https://github.com/flatironinstitute/Cryo-EM-Heterogeneity-Challenge-1/blob/efb77bc21039677c6ede3a2043dad38e6dd2281f/src/cryo_challenge/_preprocessing/align_utils.py#L118C1-L154C19)

```python
def align_submission(
    volumes: torch.Tensor, ref_volume: torch.Tensor, params: dict
) -> torch.Tensor:
    """
    Align submission volumes to ground truth volume

    Parameters:
    -----------
    volumes (torch.Tensor): submission volumes
        shape: (n_volumes, im_x, im_y, im_z)
    ref_volume (torch.Tensor): ground truth volume
        shape: (im_x, im_y, im_z)
    params (dict): dictionary containing alignment parameters

    Returns:
    --------
    volumes (torch.Tensor): aligned submission volumes
    """
    obj_vol = volumes[0].numpy().astype(np.float32)


    obj_vol = Volume(obj_vol / obj_vol.sum())
    ref_vol = Volume(ref_volume / ref_volume.sum())


    _, R_est = align_BO(
        ref_vol,
        obj_vol,
        loss_type=params["BOT_loss"],
        downsampled_size=params["BOT_box_size"],
        max_iters=params["BOT_iter"],
        refine=params["BOT_refine"],
    )
    R_est = Rotation(R_est.astype(np.float32))


    volumes = torch.from_numpy(Volume(volumes.numpy()).rotate(R_est)._data)


    return volumes
```

Create a function `align_volume` that aligns the volumes one by one, instead of aligning just one and then using the same rotation matrix for every other volume.

**Comment by DSilva27** [comment]
closed in #92
